keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side/README.md

```diff
@@ -145,6 +145,6 @@ azure:
 ```
 <!-- LINKS -->
 
-[azure_spring_boot_starter_key_vault_certificates]: https://github.com/Azure/azure-sdk-for-java/blob/azure-spring-boot_3.6.0/sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md
-[steps_to_store_certificate]: https://github.com/Azure/azure-sdk-for-java/blob/azure-spring-boot_3.6.0/sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md#creating-an-azure-key-vault
+[azure_spring_boot_starter_key_vault_certificates]: https://github.com/Azure/azure-sdk-for-java/blob/azure-spring-boot_3.14.0/sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md
```

Comment:
We need to update this when 4.x support this feature. How about creating a new issue to track this?

Reply:
There's already one https://github.com/Azure/azure-sdk-for-java/issues/28243